cluster/manifests/logging-agent/config.yaml

```diff
@@ -42,6 +55,18 @@ data:
       @type token
     </filter>
 
+    <filter scalyr-agent.**>
+      @type record_transformer
+      <record>
+        # s3_path is: ${application}/${component}/${namespace}/${environment}/${version}/${cont_name}
+        s3_path     "logging-agent/logging/visibility/{{ .Environment }}/v0.30/scalyr-agent"
```

Comment:
`{{ .Environment }}` is not expanded

Reply:
As far as I understand this is something that is expanded when it is deployed/applied

Reply:
Yes, this is expanded by the [cluster-lifecycle-manager](https://github.com/zalando-incubator/cluster-lifecycle-manager) when applying the manifests.